skills/doc-agent/references/readme-yaml.schema.yaml

```
@@ -0,0 +1,39 @@
+stack: [dotnet, blazor]
+entry_points: # leaf: where to start without LSP
+  - Mystira.App.PWA.csproj
+  - Program.cs
+skip: # do not crawl
+  - bin
+  - obj
+  - artifacts
+last_synced: '2026-08-14'
+
+# Example: index (parent directory with children)
+schema: readme-map/v1
```

YAMLlint (1.37.1)

[error] duplication of key "schema" in mapping

(key-duplicates)